content/post/benchmarking-gnns/index.md

```diff
@@ -0,0 +1,249 @@
+
+{{< figure src="imagenet_leaderboard.png" title="Fig 1: ImageNet Classification Leaderboard from [paperswithcode.com](https://paperswithcode.com)" lightbox="true" width="100%">}}
+
+Benchmarking has been proved to be beneficial for **driving progress**, identifying **essential ideas**, and solving domain-related problems in [many](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1738-8) sub-fields of science. This project was conceived with this fundamental motivation.
+
+---
+
+### Need of a benchmarking framework for GNNs
+
+#### a. Datasets:
+Many of the widely cited papers in the GNN literature contain experiments that are evaluated on **small graph datasets** which have only hundreds (or, few thousand graphs). 
+
+{{< figure src="tu_datasets.png" title="Fig 2: Statistics of the widely used TU datasets. Source [Errica et al., 2020](https://openreview.net/forum?id=HygDF6NFPB)" lightbox="true" width="100%">}}
+
+>_Take for example_, the ENZYMES dataset, which is almost seen in every work on a GNN for classification task. If one uses a random $10$-fold cross validation (in most papers), the test set would have $60$ graphs (i.e. $10$% of $600$ total graphs). That would mean a correct classification (or, alternatively a misclassification) would change $1.67$% of test accuracy score. **A couple of samples could determine a $3.33$% difference in performance measure**, which is usually a significant gain score stated when one validates a new idea in literature. You see there, the number of samples is unreliable to concretely acknowledge the advances. ^[By this, we do not mean the ideas are not useful and/or the work put by the authors is not meaningful. Every effort equally contributes to the advance of this field.]^ 
+
+Our experiments, too, show the standard deviation of performance on such datasets is large enough to actually make concrete conclusions on a research idea. Moreover, most GNNs perform statistically the same on these datasets. 
+The **quality** of these datasets also leads one to question if you should use them while validating ideas on GNNs. On several of these datasets, [simpler models](https://openreview.net/forum?id=HygDF6NFPB), sometimes, [perform as good](https://openreview.net/forum?id=rJlUhhVYvS), or even beats GNNs. 
+
+>Consequently, it has become difficult to differentiate [complex](https://arxiv.org/abs/1905.09550), [simple](https://arxiv.org/abs/1905.04579) and [graph-agnostic](https://openreview.net/forum?id=HygDF6NFPB) architectures for graph machine learning. 
```

Comment:
I would suggest not using '>' (ie. quote style) as much in the blog. For example, this line above is a good line which we may want to emphasize. Thus, you can consider styles such as https://sourcethemes.com/academic/docs/writing-markdown-latex/#asides. I would suggest the blue info bubble via "{{% alert note %}}".

Reply:
Okay! Sounds good.